common/djangoapps/track/views/__init__.py

```diff
@@ -1,6 +1,7 @@
 import datetime
 
 import pytz
+import logging
```

Comment:
nitpick:  move the import for 'logging' up, so it is below 'import datetime'.  (standard lib imports come first)